.env.example

```diff
@@ -21,4 +21,5 @@ NEXT_PUBLIC_APP_FID = '123'
 NEXT_PUBLIC_SUPABASE_URL = ''
 NEXT_PUBLIC_SUPABASE_ANON_KEY = ''
 NEXT_PUBLIC_PRIVY_API_KEY = ''
-NEXT_PUBLIC_ALCHEMY_API_KEY = ''
\ No newline at end of file
+NEXT_PUBLIC_ALCHEMY_API_KEY = ''
+NEXT_PUBLIC_SEGMENT_WRITE_KEY = ''
```

Comment:
This key is meant for client-side use and is restricted to our configured domain